modules/data.remote/inst/download_SMAP_gee2pecan.R

```diff
@@ -0,0 +1,107 @@
+##'@name download_SMAP_gee2pecan
+##'@description: 
+##'Download SMAP data from GEE by date and site location
+##'
+##'Requires python3 and earthengine-api. 
+##'Untill 'gee2pecan_smap' is integrated into PEcAn workflow,
+##'follow GEE registration 'Installation Instructions' here:
+##'https://github.com/PecanProject/pecan/pull/2645
+##'
+##'@param start  start date as YYYY-mm-dd (chr)
+##'@param end  end date YYYY-mm-dd (chr)
+##'@param site_id Bety site location id number(s)
+##'@param geoJSON_outdir  directory to store site GeoJSON, must be the location same as 'gee2pecan_smap.py'
+##'@param smap_outdir  directory to store netCDF file of SMAP data, if directory folder does not exist it will be created
+##'@return data.frame of SMAP data w/ Date, NA's filling missing data
+##'
+##'
+##'@authors Juliette Bateman, Ayush Prasad (gee2pecan_smap.py)
+##'
+##'@examples
+##'\dontrun{
+##'test <- download_SMAP_gee2pecan(
+##'  start = "2019-11-01",
+##'  end = "2019-11-10",
+##'  site_id = 676,
+##'  geoJSON_outdir = "/fs/data3/jbateman/pecan/modules/data.remote/inst", 
+##'  smap_outdir = "/fs/data3/jbateman/pecan/modules/data.remote/inst")
+##'}
+
+
+download_SMAP_gee2pecan <- function(start, end,
+                                    site_id, 
+                                    geoJSON_outdir, smap_outdir) {
+  
+  
+  #################### Connect to BETY #################### 
+  
+  bety <- list(user='bety', password='bety', host='localhost',
+               dbname='bety', driver='PostgreSQL',write=TRUE)
+  con <- PEcAn.DB::db.open(bety)
+  bety$con <- con
+  site_ID <- as.character(site_id)
+  suppressWarnings(site_qry <- glue::glue_sql("SELECT *, ST_X(ST_CENTROID(geometry)) AS lon,
+                                              ST_Y(ST_CENTROID(geometry)) AS lat FROM sites WHERE id IN ({ids*})",
+                                              ids = site_ID, .con = con))
+  suppressWarnings(qry_results <- DBI::dbSendQuery(con,site_qry))
+  suppressWarnings(qry_results <- DBI::dbFetch(qry_results))
+  site_info <- list(site_id=qry_results$id, site_name=qry_results$sitename, lat=qry_results$lat,
+                    lon=qry_results$lon, time_zone=qry_results$time_zone)
+  
+  
+  #################### Begin Data Extraction #################### 
+  
+  # Create geoJSON file for site
+  site_GeoJSON <- data.frame(site_info$lon, site_info$lat) %>%
+    setNames(c("lon","lat")) %>% 
+    leafletR::toGeoJSON(name = site_info$site_name, dest = geoJSON_outdir, overwrite = TRUE) %>%
+    rgdal::readOGR()
+  site_GeoJSON$name = site_info$site_name
+  site_GeoJSON = site_GeoJSON[-1] %>%
+    leafletR::toGeoJSON(name = site_info$site_name, dest = geoJSON_outdir, overwrite = TRUE)
+  
+  # Locate gee2pecan_smap.py function and load into R
+  script.path = file.path("/fs/data3/jbateman/pecan/modules/data.remote/inst/gee2pecan_smap.py")
```

Comment:
@julbateman you can't hard code paths like this, as this won't work for everyone else. You should use `system.file` to get the path of that specific file in that specific package wherever the package happens to be installed.